evaluation/evaluate.py

```diff
@@ -161,29 +128,76 @@ def run_evaluation_from_config(working_dir: Path, config: dict, num_questions: i
 
     openai_config = service_setup.get_openai_config()
     testdata_path = working_dir / config["testdata_path"]
-
-    evaluation_run_complete = run_evaluation(
-        openai_config=openai_config,
-        testdata_path=testdata_path,
-        results_dir=results_dir,
-        target_url=os.environ.get("BACKEND_URI") + "/ask" if target_url is None else target_url,
-        target_parameters=config.get("target_parameters", {}),
-        passing_rate=config.get("passing_rate", 3),
-        max_workers=config.get("max_workers", 4),
-        num_questions=num_questions,
-        requested_metrics=config.get(
-            "requested_metrics",
-            [
-                "gpt_groundedness",
-                "gpt_relevance",
-                "gpt_coherence",
-                "answer_length",
-                "latency",
-            ],
-        ),
+    max_workers = config.get("max_workers", 4)
+    passing_rate = config.get("passing_rate", 3)
+    target_parameters = config.get("target_parameters", {})
+    requested_metrics = config.get(
+        "requested_metrics",
+        [
+            "gpt_groundedness",
+            "gpt_relevance",
+            "gpt_coherence",
+            "answer_length",
+            "latency",
+        ],
     )
+    get_model_url = os.environ.get("BACKEND_URI") + "/getmodels" if target_url is None else target_url
+    compared_models = config.get("compared_models")
+    all_models = get_models(get_model_url)
+    for elem in compared_models:
+        if elem not in all_models:
+            logger.error(f"Requested model {elem} is not available. Available metrics: {', '.join(all_models)}")
+            return False
+
+    target_url = os.environ.get("BACKEND_URI") + "/ask" if target_url is None else target_url
```

Comment:
Similar with ``/getmodels``
```suggestion
    target_url = (os.environ.get("BACKEND_URI") if target_url is None else target_url) + "/ask" 
```